docs/kpod-tag.1.md

```diff
@@ -1,29 +1,34 @@
-## kpod-tag "1" "June 2017" "kpod"
+% kpod(1) kpod-tag - add tags to an image
+% Ryan Cole
+# kpod-tag "1" "July 2017" "kpod"
 
-# NAME
+## NAME
 kpod tag - add tags to an image
```

Comment:
I'd change this line to match the README.md

kpod tag - Add an additional name to a local image